### ISSUE TITLE
Arkmanager maybe not updating

### ISSUE THREAD

**Issue by changemenemo** [TuRz4m/Ark-docker#15]
Hi again turzam, 
I have some problems with you arkmanager installation.  Arkmanager upgrade-tools command seems to make a loop at checking update.  
I would really be able to update to the latest arkmanager commit since your version stills have the downloading mod 11111 error and it has been fixed already.

**Comment by TuRz4m** [comment]
Yep, this image is fixed on an Arkmanager tag (1.5) because it should be stable.
You can use the image turzam/ark:1.4-dev to have the latest 1.6-dev of ArkManager.

**Comment by changemenemo** [comment]
I understand that the image is fixed but could you updated to the last commit? Since it seems that the error of mod 111111 is solved on 1.5 last commit like it said here : 
https://github.com/FezVrasta/ark-server-tools/issues/558

**Comment by changemenemo** [comment]
did you check?

**Comment by changemenemo** [comment]
I have build your docker from git. 
still downloading mod 1111111 .... and sorry to say this but arkmanager in 1.5 have been updated to correct this. So could you please verify the commit of the arkmanager ?

**Comment by changemenemo** [comment]
so after verification you docker is not able to ddownload last commit and don't know why .... 
It will be best if you read this : 
https://github.com/FezVrasta/ark-server-tools/issues/558

**Comment by changemenemo** [comment]
with my changes that I propose on your git it is much more workable than before....
please turzam ^^ pay attention just a little bit to the project you do

**Comment by changemenemo** [comment]
in face the problem was deeper than expected:

your copy of arkmanager-system.cfg with the different ${...} in it , doesn't change of content even with ENV in the dockerfile. 
need to find another way to do it.

**Comment by TuRz4m** [comment]
Hi !
I'm sorry, I have a lot of work in my actual job & don't have really the time right now to make support.
But : 
 1) You can't update Ark Server Tools on the docker. It's by design : This docker image is meant to be stable and reliable so I usually update Ark Server Tools with major version (the bleeding edge is always a little bit buggy).
 2) I just updated the tag 1.4-dev with the latest version of Ark Server Tools (v1.6.2). I don't really have time to test it but, if you do a `docker pull turzam/ark:1.4-dev` and then run your docker with the image `turzam/ark:1.4-dev`, you should have the latest version.

**Comment by changemenemo** [comment]
mmmh ok thanks. no worries about your work. I think that we shoudn't fix version for arkmanager. It seems to be very stable and there are several branch : stables and betas, so I think user should be able to upgrade it ... 
If you have times during the next following days, it would be great to do a session chat maybe, I would like to ask you about several changes in the docker and several problems I encountered in the modification of your dockerfile.

**Comment by changemenemo** [comment]
by the way the environment variables of arkmanager doesn't work at all. We need to fix this issue manually

**Comment by TheInternetIsReal** [comment]
Is there a way to overrule this functionality? I'd like to keep server manager tools up to date as often as possible and I have no idea how to safely migrate to 1.4 dev.
